peeringdb/api/serializers.py

```diff
@@ -6,7 +6,7 @@
 class ContactSerializer(serializers.ModelSerializer):
     class Meta:
         model = Contact
-        fields = ["id", "role", "visible", "name", "phone", "email", "url", "network"]
+        fields = ["id", "role", "visible", "name", "phone", "email", "url", "net_id"]
```

Comment:
A nested serializer might be more appropriate here.

Reply:
Sure, I'll get that done. Shouldn't take long.